routers/web/repo/view.go

```diff
@@ -980,6 +981,11 @@ func renderHomeCode(ctx *context.Context) {
 		return
 	}
 
+	// only show licenses on repository's home page
```

Comment:
That comment doesn't make any sense to a viewer not inside this PR
```suggestion
```

Reply:
48731d439f18a28cf5184f27acdf4f02faccd3aa